jigglypuffRL/deeprl/dqn/dqn.py

```diff
@@ -112,13 +115,19 @@ def __init__(
         self.save = save_params
         self.load = load_params
         self.pretrained = pretrained
+        self.network_type = network_type
+        self.history_length = None
```

Comment:
No, `batch_size` refers to the number of states that will be processed at a time. `history_length` is used to preprocess the data to give a short history instead of a single state. So normally, we use `batch_size` number of states. With CNN, we would use `batch_size` number of `phi_states`. Which are the state histories with length `history_length`

Reply:
Okay. Sorry, got confused.